index.d.ts

```diff
@@ -161,6 +161,7 @@ declare class Shopify {
         enable: (id: number) => Promise<Shopify.IDiscountCode>;
```

Comment:
Now that you are here, can you also please fix other issues here?

- `create` parameters should be `(priceRuleId: number, params: any)`.
- `delete` parameters should be `(priceRuleId: number, id: number)`.
- `enable` and `disable` do not exist, so they can be removed.
- `get` parameters should be `(priceRuleId: number, id: number)`.
- `list` parameters should be `(priceRuleId: number)`.
- `update` should be added:
  `(priceRuleId: number, id: number, params: any) => Promise<Shopify.IDiscountCode>`.

Thank you.